infrastructure/main.tf

```diff
@@ -141,6 +146,7 @@ module "case-service" {
     FPL_LOCAL_AUTHORITY_CODE_TO_HMCTS_COURT_MAPPING     = "${data.azurerm_key_vault_secret.local_authority_code_to_hmcts_court_mapping.value}"
     FPL_LOCAL_AUTHORITY_CODE_TO_CAFCASS_MAPPING         = "${data.azurerm_key_vault_secret.local_authority_code_to_cafcass_mapping.value}"
     FPL_LOCAL_AUTHORITY_CODE_TO_SHARED_INBOX_MAPPING    = "${data.azurerm_key_vault_secret.local_authority_code_to_shared_inbox_mapping.value}"
+    FPL_LOCAL_AUTHORITY_FALLBACK_INBOX                  = "${data.azurerm_key_vault_secret.local_authority_fallback_inbox.value}"
```

Comment:
you will also need to add this secret to [cnp-flux-config](https://github.com/hmcts/cnp-flux-config) repo. Let me know if you need any help with this.

Reply:
This change has been done